README.md

```diff
@@ -562,4 +562,4 @@ Thanks also to Pablo Díez (pablodip@gmail.com) for most of the work on the firs
 
 ## Licence
```

Comment:
That's deliberate actually.  This is a British English language project, so the "Licence" form for the noun is generally applicable as here.  The filename is "LICENSE" (US English noun form) because it's an explicit reference to / instance of "The MIT License", which I'm treating a proper noun and so retaining its spelling.